### DISCUSSION TITLE
Blocking access to editing the options tab while the block set is not active

### DISCUSSION THREAD

**Discussion by ph0uad** [proginosko/LeechBlockNG#477] [Q&A] [answered]
I currently have a block set that is set to go active between 9 am and 5 pm except for the last 15 minutes of every hour. During these 15 minute breaks, I can freely edit the block set. 

Is there a way to maintain these 15 minute breaks, but make it impossible to edit the settings within them, leaving the edit hours to be exclusively outside of the 9 to 5 hours?

**Comment by proginosko** [comment]
Workaround would be to use the setting "Prevent all access to options during these time periods" under General options.

**Reply by ph0uad** [reply]
Yes, I already have this setting on, but it does not apply during the 15 minute breaks at the end of every hour. I want the websites listed to be unblocked during these 15 minute breaks, but I still want to be blocked from changing the settings during them.

**Reply by proginosko** [reply]
Sorry, I don't follow. How are you implementing the 15-minutes breaks at the end of each hour?

I'm suggesting you put `0900-1700` into that setting under General (not Advanced) options.

**Reply by ph0uad** [reply]
The time periods I have set to block is like this: `0900-0944,1000-1044,1100-1144,1200-1244,1300-1344,1400-1444,1500-1544,1600-1644,1700-1714`. Is that not how I'm supposed to do it? I would like to have it be  `0900-1700` but I don't know how else to implement the 15 minute breaks.

**Reply by proginosko** [reply, marked as answer]
No, you should leave those time periods as they are. I'm talking about a different field on the General options page:

![image](https://github.com/user-attachments/assets/bd876297-b72c-4b52-88ab-2213a7644332)

**Reply by ph0uad** [reply]
My bad. Thanks

**Reply by proginosko** [reply]
No worries!
